lms/templates/student_account/login_and_register.html

```diff
@@ -22,6 +22,7 @@
             newrelic.addPageAction('xfinished');
         }
     </%static:require_module>
+    <script type="text/javascript" src="${static.url('js/student_account/tos_modal.js')}"></script>
```

Comment:
We should make the inclusion of this Javascript file configurable by site so that we can apply this new functionality on a per-site basis.

You can do this using a `SiteConfiguration` value, like so:
```
<%!
from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
%>
% if configuration_helpers.get_value('DISPLAY_TOS_IN_MODAL_ON_REGISTRATION_PAGE', False):
<script type="text/javascript" src="${static.url('js/student_account/tos_modal.js')}"></script>
% endif
```